pkg/persistence/disk_persistence_test.go

```diff
@@ -499,3 +499,55 @@ func TestDiskPersistence_AppendToArchive(t *testing.T) {
 
 	cleanup()
 }
+
+func TestDiskPersistence_Delete(t *testing.T) {
+	diskPersistence, _ := NewDiskHandle(dataDir)
+	path := fmt.Sprintf("%s/%s", pathToCurrent, dirName1)
+
+	bytesToTest := []byte{115, 111, 109, 101, 10}
+
+	diskPersistence.Save(bytesToTest, dirName1, fileName11)
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		if err != nil {
+			t.Fatalf("Dir [%+v] was supposed to be created", path)
+		}
+	}
+
+	err := diskPersistence.Delete(dirName1, fileName11)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	if _, err := os.Stat(path); !os.IsNotExist(err) {
+		if err != nil {
+			t.Fatalf("Dir [%+v] was supposed to be moved", path)
+		}
+	}
```

Comment:
I am confused. I would expect the file to be deleted, not the directory. See that `path` points to the directory not to the file. Also s/moved/deleted.

Reply:
💯 see https://github.com/keep-network/keep-common/pull/103#discussion_r969445720